CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,93 @@
+# PrestaShop Code of Conduct
+
+English version 
+
+## Purpose 
+
+This Code of Conduct outlines how to behave to participate in the PrestaShop open source project (the “Project”) as well as prohibited behaviours and reporting procedure for damaging behaviours and/or situations. 
+
+The Code of Conduct aims to create a positive, inspiring and welcoming community, whether you are a beginner or an expert, asking questions or seeking to share your knowledge. 
+
+
+## Scope of application
+
+This Code of Conduct applies to all people contributing to the Project, especially through online contributions, whether it is repositories, forum posts or on social media. 
```

Comment:
```suggestion
This Code of Conduct applies to all people contributing to the Project, especially through online contributions, whether it is in code repositories, forum posts or on social media. 
```

Reply:
it's not only about code, but also discussions in issues and pull requests, ...